docs/development-environment.md

```diff
@@ -206,9 +206,48 @@ _If you haven't done it yet, you may need to run `yarn` before `yarn lint` for i
 
 When working on a Windows machine, you can use [Windows Subsystem for Linux](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux). You may, however, run into issues when you want to commit your changes. In this case, and if you use an IDE like PHPStorm, you can follow the recommendations in [this post](https://alex.blog/2018/02/21/guide-to-having-phpstorm-use-windows-subsystem-for-linux-git/) to have PhpStorm Use Windows Subsystem For Linux’s Git.
 
-
 ## Developing with docker
 
 We provide a standard installation of WordPress for making Jetpack development easier.
 
 You can read the details in [docker/README.md](../docker/README.md)
+
+## Standard Development/debug Tools
+
+### WP_DEBUG
+
+You should do all Jetpack development with `define( 'WP_DEBUG', true );` in your `wp-config.php`, making sure that you’re not generating any Notices or other PHP issues in your error_log.
+
+### WP-CLI
+
+Jetpack CLI is a command line interface for Jetpack, extending off of wp-cli for WordPress. You can easily modify your installation of Jetpack with a just a few simple commands. All you need is SSH access and a basic understanding of command line tools.
+
+Usage:
+
+* wp jetpack status [<full>]
+* wp jetpack module <list|activate|deactivate|toggle> [<module_name>]
+* wp jetpack options <list|get|delete|update> [<option_name>] [<option_value>]
+* wp jetpack protect <whitelist> [<ip|ip_low-ip_high|list|clear>]
+* wp jetpack reset <modules|options>
+* wp jetpack disconnect <blog|user> [<user_identifier>]
+* wp jetpack status
+* wp jetpack status [<full>]
+
+More info can be found in [Jetpack support documentation](https://jetpack.com/support/jetpack-cli/).
+
+### JETPACK_DEV_DEBUG
+
+`JETPACK_DEV_DEBUG` constant can be used to enable development mode in Jetpack. With Development Mode, features that do not require a connection to WordPress.com servers can be activated on a localhost WordPress installation for testing.
+
```

Comment:
In the paragraph above about `WP_DEBUG, you mentioned how to actually use it (`define( 'WP_DEBUG', true );` in your `wp-config.php`). Should we have something similar here for `JETPACK_DEV_DEBUG` and for `JETPACK__SANDBOX_DOMAIN`?